webservices/common/models.py

```diff
@@ -263,6 +263,16 @@ class CommitteeReports(BaseModel):
     offsets_to_operating_expenditures_period = db.Column(db.Integer)
     offsets_to_operating_expenditures_ytd = db.Column(db.Integer)
 
+    @property
+    def pdf_url(self):
+        if self.report_year is not None and self.report_year > 1994:
```

Comment:
I was going off of a comment from the FEC but, maybe I misinterpreted it. I will double check now.